Makefile

```diff
@@ -1,9 +1,9 @@
 REPO := logdna-agent-v2
 
 # The image repo and tag can be modified e.g.
-# `make build RUST_IMAGE=docker.io/rust:1.42.0`
-RUST_IMAGE_REPO ?= docker.io/rust
-RUST_IMAGE_TAG ?= 1.42
+# `make build RUST_IMAGE=docker.io/rust:1.42.0 RUST_IMAGE_TAG=latest`
+RUST_IMAGE_REPO ?= docker.io/logdna/build-images
+RUST_IMAGE_TAG ?= rust-buster-stable
```

Comment:
Do we want to block the tag by major version? something like `rust-buster-1`. In our `build-images` repo we can ship a variety of tags: `rust-buster-1`, `rust-buster-1.42`, `rust-buster-1.42.0`.

I don't know when Rust will moves towards a backwards incompatible v2, but it seems like good practice.

Reply:
I was thinking stable, beta, maybe nightly and then the named releases, eg 1.42.1, 1.46.0 etc etc. There's value in testing against current beta and stable even if we're building releases on a pinned version. It'll probably be quite a while before any version 2.0, but I wouldn't mind putting a 1 before stable/beta/nightly